src/components/ValidateConfigs.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, ReactElement, createElement } from "react";
+import { TagContainerProps } from "./TagContainer";
+import { Alert } from "./Alert";
+
+export class ValidateConfigs extends Component<TagContainerProps & { showOnError: boolean }, {}> {
+    render() {
+        const message = ValidateConfigs.validate(this.props);
+
+        if (message) {
+            const alertClassName = "widget-tag-text-alert";
+            return this.props.showOnError
+                ? createElement("div", { className: "widget-tag-text-invalid" },
+                    createElement(Alert, { bootstrapStyle: "danger", className: alertClassName, message }),
+                    this.props.children as ReactElement<TagContainerProps>
+                ) : createElement(Alert, { bootstrapStyle: "danger", className: alertClassName, message });
+        }
+
+        return this.props.children as ReactElement<TagContainerProps>;
+    }
+
+    static validate(props: TagContainerProps): string {
+        if (props.tagAttribute === "") {
+            return `The tag attribute is missing`;
+        } else if (props.tagEntity === "") {
```

Comment:
Useless required checks, simple check those should be done use setting widget.xml required attribute.

Conditional check should be done here. Like when tag limit is set bigger it should also have a message, validate if you can find the place holder text `{blabla}` etc